instance/tests/integration/integration_instance.py

```diff
@@ -263,6 +247,24 @@ def test_activity_csv(self):
         # stdout should contain 3 lines (as opposed to 2) to account for the last newline.
         self.assertEqual(len(out_lines), 3)
 
+    ''' FIXME JV : Need to add an additional test container?
```

Comment:
@bradenmacdonald I took your suggestion with 66b4f33 and replaced the single VM ephemeral database test with a multi-vm, persistent database test.